v2/emailpassword/serverless/with-aws-lambda/authorizer.mdx

```diff
@@ -0,0 +1,113 @@
+---
+id: authorizer
+title: Lambda Authorizer
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./session/serverless/with-aws-lambda/authorizer.mdx -->
+
+# Lambda Authorizer
+
+You can use a lambda similar to [step 5](./session-verification) as an Authorizer in API Gateways. This will enable you to use SuperTokens in a lambda to authorize requests to other integrations (e.g., AppSync).
+
+## 1) Add configurations and dependencies
+
+Follow [step 1](./frontend) and [step 2](./backend-config).
+
+## 2) Create a new file `index.js`
+Use the code below as the handler for the lambda. Remember that whenever we want to use any functions from the `supertokens-node` lib, we have to call the `supertokens.init` function at the top of that serverless function file. We can then use `getSession()` to get the session.
+
+```jsx title="index.js"
+let supertokens = require("supertokens-node");
+let { wrapRequest, wrapResponse } = require("supertokens-node/framework/awsLambda");
+let Session = require("supertokens-node/recipe/session");
+
+let { getBackendConfig } = require("./config");
+
+supertokens.init(getBackendConfig());
+
+exports.handler = async function(event, context) {
+    const wrappedResponse = wrapResponse(event);
+    try {
+        const session = await Session.getSession(wrapRequest(event), wrappedResponse, { sessionRequired: false });
+        if (session) {
+            return generateAllow(session.getUserId(), event.methodArn, {
+                setCookie: event.supertokens.response.cookies.join(', '),
```

Comment:
Add a comment explaining why setCookie is needed. Something like "Sometimes `getSession` adds cookies to the response, so we want to capture those and send them to the client."